judgels-backends/jophiel/jophiel-dist/init.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -ex
+
+SERVICE=$1
+
+if [[ ! -z ${SERVICE} ]]; then
```

Comment:
Logic should be as follows:

- if the command is `dbMigrate`, pass it to jophiel binary
- else, pass it to the original `init.sh`